301 tower enterprise 224 features update (#306)
seqeralabs/nf-tower-docs: docs/monitoring/overview.md

  - **Green** are successfully executed.
  - **Yellow** are successfully executed where at least one task failed with an "ignore" error strategy.
  - **Red** are successfully executed where at least one task failed with a "terminate" error strategy.
  - **Grey** are jobs that were forced to stop during execution.

> Selecting any particular run from the panel will display that run's execution details.

Author: ```suggestion
Select any run from the panel to view that run's execution details.
```

### All runs view

> **Available from version 22.4.X**

Author: ```suggestion
**Available from version 22.4.0**
```
